src/cli.ts

```diff
@@ -185,6 +185,7 @@ program
     'the target Kibana spaces for the pushed monitors — spaces help you organise pushed monitors.'
   )
   .option('-y, --yes', 'skip all questions and run non-interactively')
+  .option('-f, --force', 'Ignore warnings and force push')
```

Comment:
I would reuse the `-y`, we already use this for ignoring some of the users changes on local and forcefully pushing the project monitors. Lets not introduce two things to do the same thing.

Reply:
Thinking through this again. This would end up costing user and doing harm than good. Lets just error in case of skipped journey and figure out a solution if there is a need.

Reply:
i have remove the -f